docs/ios/deployment/overview.md

```diff
@@ -0,0 +1,193 @@
+1. In Xcode, select the **Xcode > Preferences...** menu item.
+1. In the **Preferences** dialog, select the **Accounts** tab.
+1. In the **Accounts** tab, click the **+** button to add your Apple Developer Account to Xcode:
+
+    :::image type="content" source="media/overview/accounts-dialog.png" alt-text="Xcode Accounts dialog in preferences.":::
+
+1. In the account type popup, select **Apple ID** and then click the **Continue** button:
+
+    :::image type="content" source="media/overview/account-type.png" alt-text="Xcode select the type of account you'd like to add popup.":::
+
+1. In the sign in popup, enter your Apple ID and click the **Next** button.
+1. In the sign in popup, enter your Apple ID password and click the **Next** button:
+
+    :::image type="content" source="media/overview/sign-in.png" alt-text="Xcode Apple account sign-in.":::
+
+1. In the **Accounts** tab, click the **Manage Certificates...** button to ensure that your distribution certificate has been downloaded.
+1. In the **Accounts** tab, click the **Download Manual Profiles** button to download your provisioning profiles:
+
+    :::image type="content" source="media/overview/account-details.png" alt-text="Xcode Apple Developer Program account details.":::
+
+1. Close Xcode.
+
+## Add entitlements to your app
+
+In iOS, apps run in a sandbox that provides a set of rules that limit access between the app and system resources or user data. *Entitlements* are used to request the expansion of the sandbox to give your app additional capabilities. Any entitlements used by your app must be specified in an entitlements file. For more information about entitlements, see [Entitlements](entitlements.md).
+
+## Add code signing data to your app project
+
+There are project-level settings you must set to sign your iOS app with its provisioning profile. These settings are configured in a `<PropertyGroup>` node:
+
+- `<RuntimeIdentifier>` – the runtime identifier (RID) for the project. Set to `ios-arm64`.
+- `<CodesignKey>` – the name of the distribution certificate you installed into Keychain Access on your Mac build host.
+- `<CodesignProvision>` – the provisioning profile name.
+- `<CodesignEntitlement>` – the name of the entitlements file. Set to `Entitlements.plist`. This setting need only be specified if you're using entitlements.
+- `<ArchiveOnBuild>` – a boolean value that indicates whether to build the app package. Set to `true`.
+- `<TcpPort>` – the TCP port on which to communicate with your Mac build host. Set to `58181`.
+- `<ServerAddress>` – the IP address of your Mac build host.
+- `<ServerUser>` – the username to use when logging into your Mac build host. Use your system username rather than your full name.
+- `<ServerPassword>` – the password for the username used to log into the Mac build host.
+- `<_DotNetRootRemoteDirectory>` – the folder on the Mac build host that contains the .NET SDK. Set to `/Users/{macOS username}/Library/Caches/Xamarin/XMA/SDKs/dotnet/`.
```

Comment:
Personal note: did you verify this is actually (still) needed? Seemed to be needed when I looked into it, but hoped that would not be necessary later on :D

Reply:
I did not. Will try it without now.

Reply:
It's still required :(